main.cpp

```diff
@@ -8,12 +8,15 @@
 int main(int argc, char *argv[])
 {
     char optc;
+    bool invert_steering = false;
+    bool invert_throttle = false;
+    bool invert_axis = false;
 
     rf_global_args.FREQ = DEFAULT_FREQ;
     rf_global_args.SAMPLE_RATE = DEFAULT_SAMPLE_RATE;
     rf_global_args.SYMBOL_RATE = DEFAULT_SYMBOL_RATE;
 
-    while ((optc = getopt(argc, argv, "f:s:b:")) > 0) {
+    while ((optc = getopt(argc, argv, "f:s:b:STA")) > 0) {
```

Comment:
Could you please explain what is the use case for swapping axes and inverting throttle/steering?